src/utils/common-utils/common-utils.js

```diff
@@ -1,3 +1,14 @@
 export const deepCloneJson = (objectOrArray) => {
   return JSON.parse(JSON.stringify(objectOrArray));
 };
+
+export const focusErrors = (firstSummaryError) => {
+  if (firstSummaryError) {
+    firstSummaryError.scrollIntoView({
```

Comment:
Wow I know it’s copy/pasted code, but TIL about element.scrollIntoView, this is great 😄